libs/langchain/langchain/schema/runnable/base.py

```diff
@@ -1428,16 +1428,27 @@ class RunnableMap(RunnableSerializable[Input, Dict[str, Any]]):
 
     def __init__(
         self,
-        steps: Mapping[
-            str,
-            Union[
-                Runnable[Input, Any],
-                Callable[[Input], Any],
-                Mapping[str, Union[Runnable[Input, Any], Callable[[Input], Any]]],
-            ],
+        __steps: Optional[
+            Mapping[
+                str,
+                Union[
+                    Runnable[Input, Any],
+                    Callable[[Input], Any],
+                    Mapping[str, Union[Runnable[Input, Any], Callable[[Input], Any]]],
+                ],
+            ]
+        ] = None,
+        **kwargs: Union[
```

Comment:
doesn't **kwargs have to be mapping?

Reply:
Strangely the annotation you attach to kwargs is actually the annotation for the value of the mapping. Same with *args